app/views/shared/_suggestion.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="container">
+	<% @music.each_with_index do |tune, index| %>
+	<div class="row pairing">
+		<div class="col-md-1 fav-button">
+			<% if current_user %>
+			<% if @favorites.include?(@results[index]["id"]) %>
+			<%= link_to "Unfave!", favorite_path(@results[index]["id"]), method: :delete, class: "btn btn-danger" %>
+			<% else %>
+			<%= link_to "Fave!", favorite_path(@results[index]["id"]), method: :post, class: "btn btn-primary" %>
+			<% end %>
+			<% end %>
+		</div>
+		<div class="col-md-6 music-food">
+			<iframe src="https://embed.spotify.com/?uri=<%= tune.uri %>" width="300" height="80" frameborder="0" allowtransparency="true"></iframe><br>
+			<%= link_to tune.name, tune.external_urls["spotify"], class: "body-links" %><br>
+			<%= tune.type.capitalize %><br>
```

Comment:
In class, we were told avoid using the `<br>` tag and instead opt for styling with CSS. In this example, these could be items in a list.